example_configs/nextcloud_oidc_authelia.md

```diff
@@ -0,0 +1,81 @@
+## Authelia
+
+Set up Authelia according to it's [documentation](https://www.authelia.com/overview/prologue/introduction/), including the [OpenID Connect](https://www.authelia.com/configuration/identity-providers/open-id-connect/) and [Nextcloud instructions](https://www.authelia.com/integration/openid-connect/nextcloud/).
+
+## LLDAP
+
+Set up [LLDAP (Light Lightweight Directory Access Protocol](https://github.com/nitnelave/lldap) according to its standard documentation. 
+
+Add a group and note the name you use. For this tutorial, we're using the group `users`. Create a new user and add it to the `users` group. 
+
+#### Optional:
+Once setup, add an admin or config user and add to the `lldap_strict_readonly` group. This will be the config account used for Nextcloud to read your groups and users from the server. 
+
+## Nextcloud
+**_When we get to the OpenID section, we will be using the same defaults as Authelia's documentation. As a reminder, they are:_** 
+
+* **Application Root URL:** https://nextcloud.example.com
+* **Authelia Root URL:** https://auth.example.com
+* **Client ID:** nextcloud
+* **Client Secret:** nextcloud_client_secret
+
+Login to your Nextcloud instance as an admin. Setup an LDAP server connection according to [LLDAP's Documentation](https://github.com/nitnelave/lldap/blob/main/example_configs/nextcloud.md).Once your configuration is setup and Nextcloud doesn't show any errors, you can move onto the next step. It's also a good idea to test this by logging into your Nextcloud instance with your LLDAP credentials. OpenID Connect won't work if you don't first have this working. 
+
+Next, navigate to the `Apps` section. 
+
+[![nextcloud_apps.png](images/nextcloud_apps.png)
```

Comment:
Can you add the images?

Reply:
And you've got an extra "[" on this line.
Tip: On GitHub, on the PR screen, you can click the three dots in the upper right corner, then "View this file" to see the markdown rendered. You'll see the image missing, the formatting and so on.

Reply:
I previewed and thought I got all the stray "[" but I see I missed some. Also, I updated the images and previewed the markdown locally, which pulled and loaded from that dir as expected. Are the images not being uploaded? I'm now thinking I Pr'd the wrong branch? Let me check over it again and I'll comment here or message you on discord.

Reply:
You can see the "Files changed" in the PR: there are only the two markdown files, no new images.

Reply:
I think I know what I did wrong. I'll finish my edits and push changes. I bet I only pushed the two files.